extractor/src/test/java/org/schabi/newpipe/extractor/utils/UtilsTest.java

```diff
@@ -15,4 +19,10 @@ public void testMixedNumberWordToLong() throws JsonParserException, ParsingExcep
         assertEquals(10.5e6, Utils.mixedNumberWordToLong("10,5M"), 0.0);
         assertEquals(1.5e9, Utils.mixedNumberWordToLong("1,5B"), 0.0);
     }
+
+    public static void createFile(String path, String content) throws IOException {
```

Comment:
No. I'm using often a function of this kind when testing things in NewPipeExtractor.
I thought it could be useful for everyone and then I put it here.